sos/report/__init__.py

```diff
@@ -1402,7 +1472,7 @@ def execute(self):
                 return False
 
             self.add_manifest_data()
-            self.batch()
+            self.batch(self._check_estimate_mode())
```

Comment:
Rather than passing this as a parameter, I think it'd be better to call `_check_estimate_mode()` inside `batch()`, e.g.:

```python
def batch(self):
    msg = self.policy.get_msg()
    if self.opts.estimate_mode:
        msg += self._check_estimate_mode()
```